src/Symfony/Component/Form/Form.php

```diff
@@ -580,6 +580,8 @@ public function submit($submittedData, $clearMissing = true)
                         if (method_exists($child, 'getClickedButton') && null !== $child->getClickedButton()) {
                             $this->clickedButton = $child->getClickedButton();
                         }
+                    } elseif (!$isSubmitted && !$clearMissing && array_key_exists($name, $submittedData)) {
```

Comment:
I know, this is a dummy test, see https://github.com/HeahDude/symfony/commit/935b2da6b8bad1a9c2d8174d0b3788fc10070183#diff-ca5e25b47f3ecc94cd557946aeb486c6R583 for the right fix